external/iplib/asyncio/base_events.py

```
@@ -0,0 +1,1338 @@
+            # Compute the desired timeout.
+            when = self._scheduled[0]._when
+            timeout = max(0, when - self.time())
+
+        if self._debug and timeout != 0:
+            t0 = self.time()
+            event_list = self._selector.select(timeout)
+            dt = self.time() - t0
+            if dt >= 1.0:
+                level = logging.INFO
+            else:
+                level = logging.DEBUG
```

Ruff (0.8.2)

Use ternary operator `level = logging.INFO if dt >= 1.0 else logging.DEBUG` instead of `if`-`else`-block

Replace `if`-`else`-block with `level = logging.INFO if dt >= 1.0 else logging.DEBUG`

(SIM108)